docs/services/vision/classification.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Classification (or 2D image classification)"
+linkTitle: "Classification"
+weight: 20
+type: "docs"
+description: "Select an algorithm that outputs a class label and confidence score associated with a 2D image."
+tags: ["vision", "computer vision", "CV", "services", "classification"]
+# SMEs: Bijan, Khari
+---
+
+_2D Image Classification_ is the process of taking a 2D image from a camera and deciding which class label, out of many, best describes the given image.
+Any camera that can return 2D images can use 2D image classification.
+
+Which class labels may be considered for classification varies and will depend on the machine learning model and how it was trained.
+
+The returned classifications consist of the image's class label and confidence score.
+
+* `class_name` (string): specifies the label of the found object.
+* `confidence` (float): specifies the confidence of the assigned label.
+  Between `0.0` and `1.0`, inclusive.
+
+## Classifier Types
+
+The types of classifiers supported are:
+
+* **tflite_classifier**: a machine-learning based classifier that returns a class label and confidence score according to the specified `tensorflow-lite` model file available on the robot’s hard drive.
```

Comment:
```suggestion
* **tflite_classifier**: a machine-learning-based classifier that returns a class label and confidence score according to the specified `tensorflow-lite` model file available on the robot’s hard drive.
```
or perhaps
```suggestion
* **tflite_classifier**: a machine learning-based classifier that returns a class label and confidence score according to the specified `tensorflow-lite` model file available on the robot’s hard drive.
```
Machine learning wouldn't be hyphenated if it weren't part of the compound adjective (with "based"), I think